### DISCUSSION TITLE
Updating parameters for Rodauth routes

### DISCUSSION THREAD

**Discussion by mdodell** [janko/rodauth-rails#135] [Q&A] [answered]
My apologies for all of the questions! Integrating the library in our new project, it's absolutely amazing. I was astounded at how quickly I was able to get fully-fledged authentication with Omniauth up and running.

Is it possible to update the casing of parameters? For example, in the case of `create-account`, turning:
```
{
    "login": "myemail@gmail.com",
     "password": "confirm",
     "password-confirm": "confirm"
}
```

to:
```
{
    "login": "myemail@gmail.com",
     "password": "confirm",
     "passwordConfirm": "confirm"
}
```

Same thing with any response property keys!

**Comment by janko** [comment, marked as answer]
Yes, this is added by the [login_password_requirements_base](http://rodauth.jeremyevans.net/rdoc/files/doc/login_password_requirements_base_rdoc.html) feature, which allows configuration:

```rb
password_confirm_param { "passwordConfirm" }
```
